hubspot/serializers.py

```diff
@@ -146,8 +160,15 @@ def get_price(self, instance):
         """Return the latest product version price"""
         product_version = instance.latest_version
         if product_version:
-            return product_version.price.to_eng_string()
-        return "0.00"
+            return product_version.price
```

Comment:
This throws an error `TypeError("Object of type 'Decimal' is not JSON serializable",)`, if you revert this function back to the previous code it should work.

Reply:
Ditto for prices in other serializers

Reply:
Can this be a float? If we leave it as a string we lose the ability to use mathematical operations on the hubspot app.

Reply:
It seems like hubspot is converting it anyway?  I placed an order without a coupon for $232 this morning and the price is showing up correctly.  But if there are other issues I'm not seeing then a float is fine too as long as the serializer accepts it and it's correctly rounded to 2 decimal places.